pkg/db/db.go

```diff
@@ -0,0 +1,37 @@
+package db
+
+import (
+	"database/sql"
+	"errors"
+	"os"
+
+	"github.com/fuckingUnicorn/final_project/pkg/constants"
+
+	
+
+	_ "modernc.org/sqlite"
+)
+
+var db *sql.DB
+
+func Close() error {
+	if db != nil {
+		return db.Close()
+	}
+	return nil
+}
+
+func Init(dbFile string) error {
+	_, err := os.Stat(dbFile)
+
+	db, err = sql.Open("sqlite", dbFile)
+	if err != nil {
+		return errors.New("error opening database: " + err.Error())
```

Comment:
Нужно оборачивать ошибку, а не создавать новую.
```suggestion
		return fmt.Errorf("error opening database: %w", err)
```